### ISSUE TITLE
Pre-Relaunch TODOs

### ISSUE THREAD

**Issue by AnonJr** [AnonJr/AnonJr.com#1]
Lost focus on the relaunch for a minute there, time to collect all the little things that need to be done before the official relaunch.

- [ ] Fix the sidebar logos for other places to find me, the SO icon is a little taller than the other two for some reason.
- [ ] Fix the tag format - those things are butt ugly.
- [ ] Fix the meta layout on the posts - need a prettier author, date/time, tag list on the byline.
- [ ] Less-busy background.
- [ ] Actually fill out the "About" page...
- [ ] Review the overall layout/structure. I'd like to re-design, but how much of that is immediate need and how much is procrastination?

**Comment by AnonJr** [comment]
Still need to re-start this... and maybe tweak the colour palate to https://www.happyhues.co/palettes/13

**Comment by AnonJr** [comment]
I'd already done most of these, and I don't remember the impetus behind the rest...
